### PR TITLE
Fklearn version 2.2.0 - changelog

### PR DESCRIPTION
**Pull request by nicolas-behar**
### Status
**READY**

### Todo list
- [x] Documentation

### Description of the changes proposed in the pull request
Pull requesto to produce a new release with the following inclusions/modificatiions:
 - Changing reference style for causal S-Learner (Readthedocs visualization)
 - Added S-Learner to the changelog
 - Added T-Learner to the changelog
 - Added bugfix to the changelog

### Related PRs
- https://github.com/nubank/fklearn/pull/201
- https://github.com/nubank/fklearn/pull/206
- https://github.com/nubank/fklearn/pull/208

### CODE REVIEW

**Review by GiulioCMSanto — COMMENTED** [review]
Hey @nicolas-cardeira , the T-learner was just merged, including the .rst for the cate_learning. Can you please make a merge with the master branch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+  - Fix validator behavior when receiving data containing gaps and a time based split function that
+    could generate empty 
+    training and testing folds and then break. 
+    The argument `drop_empty_folds` can be set to `True` to drop invalid folds from validation and 
+    store them in the 
+    log.
+- **Documentation**
+  - Including Classification S-learner documentation and changing validator documentation to 
+    reflect changes.
+
```
Comment (GiulioCMSanto) [inline CHANGELOG.md:16]: Maybe we could also include the T-learner in this release, which was just merged 😄

Comment (nicolas-behar) [reply CHANGELOG.md:16]: will do!

**Review by GiulioCMSanto — APPROVED** [review]
🚀 LGTM!

**Pull request merged**